pages/toggle-button/style-permutations.page.tsx

```diff
@@ -0,0 +1,162 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+
+import ToggleButton, { ToggleButtonProps } from '~components/toggle-button';
+
+import createPermutations from '../utils/permutations';
+import PermutationsView from '../utils/permutations-view';
+import ScreenshotArea from '../utils/screenshot-area';
+
+const permutations = createPermutations<ToggleButtonProps>([
+  {
+    variant: ['normal'],
+    children: ['Subscribe'],
+    pressed: [false, true],
+    disabled: [false, true],
+    iconName: ['thumbs-up'],
+    pressedIconName: ['thumbs-up-filled'],
+    onChange: [() => {}],
+    style: [
+      {
```

Comment:
To improve readability, I recommend extracting styles to some named variables, like "borderlessStyle" or "contrastyStyle" etc. This will improve two things:
1. give the overview of the different styles that we use (maybe we don't need that many?)
2. make it easier to comprehend the diff in between permutations